packages/theme-a11y/theme-a11y.js

```diff
@@ -5,66 +5,102 @@
  */
 
 /**
- * For use when focus shifts to a container rather than a link
+ * Moves focus to an HTML element
  * eg for In-page links, after scroll, focus shifts to content area so that
- * next `tab` is where user expects if focusing a link, just $link.focus();
+ * next `tab` is where user expects. Used in bindInPageLinks()
+ * eg move focus to a modal that is opened. Used in trapFocus()
+ *
+ * @param {Element} container - Container DOM element to trap focus inside of
+ * @param {Object} options - Settings unique to your theme
+ * @param {string} options.className - Class name to apply to element on focus.
  */
-export function pageLinkFocus(element, config) {
-  config = config || {};
+export function forceFocus(element, options) {
+  options = options || {};
 
-  var className = config.className || 'js-focus-hidden';
+  var className = options.className ? options.className : null;
   var savedTabIndex = element.tabIndex;
 
   element.tabIndex = -1;
   element.dataset.tabIndex = savedTabIndex;
   element.focus();
-  element.classList.add(className);
+  if (className) {
+    element.classList.add(className);
+  }
   element.addEventListener('blur', callback);
 
   function callback(event) {
     event.target.removeEventListener(event.type, callback);
 
     element.tabIndex = savedTabIndex;
     delete element.dataset.tabIndex;
-    element.classList.remove(className);
+    if (className) {
+      element.classList.remove(className);
+    }
   }
 }
 
 /**
  * If there's a hash in the url, focus the appropriate element
+ * This compensates for older browsers that do not move keyboard focus to anchor links.
+ * Recommendation: To be called once the page in loaded.
+ *
+ * @param {Object} options - Settings unique to your theme
+ * @param {string} options.className - Class name to apply to element on focus.
+ * @param {string} options.ignore - Selector for elements to not include.
  */
 
-export function focusHash() {
+export function focusHash(options) {
+  options = options || {};
   var hash = window.location.hash;
   var element = document.getElementById(hash.slice(1));
   // is there a hash in the url? is it an element on the page?
 
+  // if we are to ignore this element, early return
+  var ignoreSelector = options.ignore || false;
+  if (element && ignoreSelector && element.matches(ignoreSelector)) {
+    return false;
+  }
+
   if (hash && element) {
-    pageLinkFocus(element);
+    forceFocus(element, options);
   }
 }
 
 /**
  * When an in-page (url w/hash) link is clicked, focus the appropriate element
+ * This compensates for older browsers that do not move keyboard focus to anchor links.
+ * Recommendation: To be called once the page in loaded.
+ *
+ * @param {Object} options - Settings unique to your theme
+ * @param {string} options.className - Class name to apply to element on focus.
```

Comment:
Since `options` is an object containing properties, can you separate the documentation?